src/pymordemos/elliptic2.py

```diff
@@ -6,18 +6,22 @@
 """Simple demonstration of solving the Poisson equation in 2D using pyMOR's builtin discretizations.
 
 Usage:
-    elliptic2.py [--fv] PROBLEM-NUMBER N
+    elliptic2.py [--fv] PROBLEM-NUMBER N PRODUCT-COUNT
 
 Arguments:
-    PROBLEM-NUMBER    {0,1}, selects the problem to solve
-    N                 Triangle count per direction
+    PROBLEM-NUMBER  {0,1}, selects the problem to solve
+    N               Triangle count per direction
+
+    PRODUCT-COUNT   0: equip the model with standard products
+                    1: add an energy product for a fixed parameter
```

Comment:
I don't think the user will know what that is supposed to mean.

Reply:
Also, which parameter values?

Reply:
I tried to be more precise in https://github.com/pymor/pymor/pull/1008/commits/80285ba4b3900fdb641726773f1de98938fc9aa4. Not sure whether this is what you were thinking about